modules/azure-landing-zone/frontdoor.tf

```diff
@@ -211,3 +191,39 @@ resource "azurerm_frontdoor" "main" {
 
   depends_on = [azurerm_frontdoor_firewall_policy.custom, azurerm_key_vault_access_policy.frontdoor_kv_access]
 }
+
+resource "azurerm_frontdoor_custom_https_configuration" "https" {
+  for_each = { for frontend in var.frontends :
+    frontend.name => frontend
+    if lookup(frontend, "enable_ssl", true)
+  }
+
+  resource_group_name               = azurerm_frontdoor.main.resource_group_name
+  frontend_endpoint_id              = "/subscriptions/${var.subscription_id}/resourcegroups/${azurerm_frontdoor.main.resource_group_name}/providers/Microsoft.Network/frontdoors/${azurerm_frontdoor.main.name}/frontendendpoints/${each.value["name"]}"
```

Comment:
Shouldn't have to create the id. 
`  frontend_endpoint_id              = azurerm_frontdoor.example.frontend_endpoint[1].id
`